src/components/BookList.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Book from './Book';
+import BookForm from './BookForm';
+
+const BookList = () => (
+  <>
+    <Book title="Learn React" author="Kent C Doods" />
+    <Book title="Learn React" author="Kent C Doods" />
+    <BookForm />
+  </>
+);
+
+export default BookList;
```

Comment:
- [ ] [REQUIRED] Great job with the implementations @Nyame-Wolf  👍🏽 . However, a part of the project requirement stated that you need to add a button to remove books. Kindly fix this to get this project approved.